collector-sqs/src/main/java/zipkin/collector/sqs/SQSCollector.java

```diff
@@ -136,22 +143,24 @@ public Builder parallelism(int parallelism) {
 
     final String queueUrl;
     final int parallelism;
-    final AmazonSQSAsync client;
+    final AmazonSQS client;
     final Collector collector;
     final int waitTimeSeconds;
+    final int maxNumberOfMessages;
 
-    LazyProcessors(AmazonSQSAsync client, Collector collector, String queueUrl, int parallelism, int waitTimeSeconds) {
+    LazyProcessors(AmazonSQS client, Collector collector, String queueUrl, int parallelism, int waitTimeSeconds, int maxNumberOfMessages) {
```

Comment:
note, a little less maintenance to pass the builder as the ctor arg and then copy the values from it as you do below